### ISSUE TITLE
Cannot add documents (only in a dev branch): unsupported snappy compression

### ISSUE THREAD

**Issue by curquiza** [meilisearch/meilisearch#2251]
⚠️ This bug is only on this [branch](https://github.com/meilisearch/meilisearch/pull/1928), but this branch is used by the @meilisearch/cloud-team in their dev SaaS via the `v0.26.1-cloud-team` tag

It's impossible to add documents, getting an internal error.

Steps:
- Add documents
```bash
curl \
  -X POST 'http://127.0.0.1:7700/indexes/movies/documents' \
  -H 'Content-Type: application/json' \
  --data-binary '[{"id": 1}]'
```
- Check the task status and get
```
{
      "details": {
        "indexedDocuments": 0,
        "receivedDocuments": 1
      },
      "duration": "PT0.001297S",
      "enqueuedAt": "2022-03-19T15:08:41.940767Z",
      "error": {
        "code": "internal",
        "link": "https://docs.meilisearch.com/errors#internal",
        "message": "unsupported snappy compression",
        "type": "internal"
      },
      "finishedAt": "2022-03-19T15:08:41.942588Z",
      "indexUid": "movies",
      "startedAt": "2022-03-19T15:08:41.941291Z",
      "status": "failed",
      "type": "documentAddition",
      "uid": 2
    }
```

**Comment by Kerollmops** [comment]
Thank you @curquiza for this issue, it seems like the issue comes from this line that correspond to the arguments used by Meilisearch and that were exposed in the latest v0.26.1-cloud-team release.

https://github.com/meilisearch/meilisearch/blob/b2bbd13c27c5719e5aea53fc60815975b5aa422b/meilisearch-lib/src/options.rs#L35

We must remove the _default value_ setting. This way the `Default` implementation is used and `CompressionType::None` is used by the engine (which is expected).

By the way this argument can be removed without consequences. To be checked.

**Comment by curquiza** [comment]
Fixed on #1928 💪
